docs/header.html

```diff
@@ -0,0 +1,9 @@
+<div class="branding">
+    <h1 class="logo">
+        <a href="index.html">
+            <img src="assets/images/logo.png" width="180"
+                 height="50">
+        </a>
+        <a class="btn btn-primary btn-cta" style="float: right;margin-top: 10px;" href="contribute.html"> Contact US</a>
+    </h1>
+</div>
```

Comment:
Curious question: Can't we move the IE version and google analytics code also in header?

Observed that in some pages IE8 and IE9 are checked whereas in some it is not?